adapters/factMapper/builder_test.go

```diff
@@ -26,17 +26,20 @@ func TestBuilderInvariants(t *testing.T) {
 	testutil.TestBuilderInvariants(b, t)
 }
 
-func TestNoRules(t *testing.T) {
+func setup(t *testing.T, bc BuilderConfig, ac AdapterConfig) adapters.FactConverter {
```

Comment:
I actually added more stuff into setup. I like having the redundant bits all in one place, especially since these bits are not really the object of the individual tests